src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -260,12 +260,7 @@ public override BoundNode VisitLambda(BoundLambda node)
                     _instrumenter = RemoveDynamicAnalysisInjectors(oldInstrumenter);
                 }
 
-                var visited = (BoundLambda)base.VisitLambda(node)!;
-                if (RewriteNullChecking(visited.Body) is BoundBlock newBody)
```

Comment:
>RewriteNullChecking [](http://example.com/codeflow?start=20&length=19)

Where does is null checking added for lambdas with this change? In `MethodCompiler.CompileSynthesizedMethods()`?

Reply:
Yes.